dod.py

```diff
@@ -55,3 +75,42 @@
 pprint.pprint(us_only.head())
 
 us_only.to_csv("results/counties.csv")
+
+states_group = us_only.groupby(['Province/State'])
+states_agg = states_group.aggregate({
+    'Last Update': 'max',
+    'Confirmed': 'sum',
+    'Recovered': 'sum',
+    'Deaths': 'sum',
+    'Active': 'sum',
+    'Country/Region': 'first',
+    'Latitude': 'first',
+    'Longitude': 'first'
+    # People tested is currently null
+    #'People Tested': 'sum'
+})
+states_agg.info()
+states_abbrev = states_agg.merge(
+    abbrev_df, left_index=True, right_on='state', how='left'
+).merge(
+    # inner merge to filter to only the 50 states
+    interventions_df, left_on='abbreviation', right_on='state', how='inner'
+).merge(
+    projections_df, left_on='state_y', right_on='State', how='left'
+).drop(['abbreviation', 'state_y', 'State'], axis=1)
+state_col_remap = {
+    'state_x': 'Province/State',
+    'Projection1': '4-day Hospitalizations Prediction',
+    'Projection2': '8-day Hospitalizations Prediction',
+    'intervention': 'Intervention'
+}
+states_remapped = states_abbrev.rename(columns=state_col_remap)
+states_remapped.info()
+# TODO: filter out county-specific columns
+state_cols = cols + ['Intervention', '4-day Hospitalizations Prediction', '8-day Hospitalizations Prediction']
```

Comment:
to me, this looks like there is a standard interface for these columns, and that interface should be seperated from the logic above